pkg/seshttp/middleware.go

```diff
@@ -84,6 +85,7 @@ func sessionCookie(sessionKey string, secure bool) *http.Cookie {
 		Value:    sessionKey,
 		HttpOnly: true,
 		Path:     "/",
+		SameSite: http.SameSiteLaxMode,
```

Comment:
Why does this need to be lax for my move?

Reply:
According to the comment in the repo: `'strict' breaks the use of the login.gov redirect`